### ISSUE TITLE
A warning

### ISSUE THREAD

**Issue by nmvdw** [UniMath/UniMath#1791]
We get the following warning

> Warning: Warning: Overwriting previous delimiting key cat in scope cat
[overwriting-delimiting-key,parsing,default]

**Comment by nmvdw** [comment]
It happens when importing `Require Import UniMath.CategoryTheory.Core.Categories.`

**Comment by benediktahrens** [comment]
Is this new, eg, caused by a new version of Coq?

**Comment by nmvdw** [comment]
Yes, I think. My guess is that some scoping is handled differently, so we need to update the code.

**Comment by rmatthes** [comment]
I am trying to solve this problem - the number of warnings for this problem is overwhelming.
